contracts/lib/platform/ERCDotFactory.sol

```diff
@@ -30,9 +30,9 @@ contract ERCDotFactory is Ownable {
         bytes32 providerTitle,
         bytes32 specifier,
         bytes32 symbol,
-        int256[] curve
-    ) returns(address) {
-        require(curves[specifier] == 0, "Curve specifier already exists");
+        int256[] memory curve
+    ) public returns(address) {
+        require(curves[specifier] == address(0) , "Curve specifier already exists");
```

Comment:
require(curves[specifier] == 0) .         could not compare and address to a uint, so casted address to zero with address(0)